File: tt2/Cometa.java
public class Cometa {
    public static void main(String[] args){
        int ano, vezes;

        ano = MyIO.readInt();
        
        vezes = (ano - 10 ) / 76 ;
        vezes++;

        MyIO.println( +vezes * 76 + 10);
    }
}

File: tt2/Cometa.c
#include <stdio.h>

int  main (void) {
	int ano, vezes;

	scanf( "%d", &ano);
	
	vezes = (ano - 10 ) / 76 ;
	vezes++;
	printf( "%d", vezes * 76 + 10 );
	
	return  0 ;
}